### PR TITLE
fixed XAPID-981, permission denied bug

### PR DESCRIPTION
**Pull request by haomingzhang**

### CODE REVIEW

File: pgoutput/Dockerfile [hunk 1]
```diff
@@ -4,7 +4,8 @@ RUN mkdir /src
 COPY . /src/pgoutput
 # Contents of "scripts" fix pg_hba.conf and postgresql.conf
 COPY scripts /docker-entrypoint-initdb.d
-
+RUN chmod +r /docker-entrypoint-initdb.d/*
```
Comment (alexkhimich) [inline pgoutput/Dockerfile:7]: How come the previous versions were working without this?

Comment (alexkhimich) [inline pgoutput/Dockerfile:7]: ^^^6

Comment (haomingzhang) [reply pgoutput/Dockerfile:7]: No idea. maybe docker version, or maybe Gubuntu put some restrictions on it.

Comment (haomingzhang) [reply pgoutput/Dockerfile:7]: Do you suggest `chmod +6` or `chmod 666`?

Comment (gbrail) [reply pgoutput/Dockerfile:7]: That works for me. "a+r" makes it so everyone can read. Works great.

So LGTM and I'm going to merge this.

It'd be fun to unravel the mystery of why this breaks, but for another time...

Comment (sramamoorthy73) [reply pgoutput/Dockerfile:7]: I am going to try out. Will keep you posted.

**Pull request merged**